.gitlab-ci.yml

```diff
@@ -31,6 +31,7 @@ include:
       - $CI_PIPELINE_SOURCE == "push"
   variables:
     ENVIRONMENT_URL: https://www-dev.allizom.org
+    BOUNCER_URL: https://www-dev.allizom.org
```

Comment:
Shouldn't this be the URL for staging bouncer?

Reply:
palm of the face. i need to slow down.

updating now